proto/process/agent.proto

```diff
@@ -1292,3 +1312,8 @@ message RouteMetadata {
 	int32 tagIndex = 2;
 	int64 tagsModified = 3;
 }
+
+// replaces DNSEntry.  Is a list of DNS names that a given IP resolved to.
+message DNSDatabaseEntry {
+	repeated int32 nameIndexes = 1;
```

Comment:
Could we add a comment explaining what this is ? `replaces DNSEntry` isn't very helpful because you need to already know what DNSEntry is. 


I suggest:

```suggestion
        // a list of incides into CollectorConnections#encodedDomainDatabase. Use  <name of module> to unpack the domain names. 
	repeated int32 nameIndexes = 1; 
```